server/db/models/orderitem.js

```diff
@@ -17,9 +17,12 @@ const OrderItem = db.define('orderitem', {
   price: {
     type: Sequelize.DECIMAL(10, 2),
   },
-  // productId: {
-  //   type: Sequelize.UUID,
-  // },
+  productId: {
```

Comment:
If there is a relationship between OrderItem and Product and Order... do we need to have this in the OrderItem model?